tools/myst/host/exec_linux.c

```diff
@@ -57,6 +57,12 @@ Options:\n\
     --app-config-path <json> -- specifies the configuration json file for\n\
                                 running an unsigned binary. The file can be\n\
                                 the same one used for the signing process.\n\
+    --host-to-enc-uid-map <host-uid:enc-uid[,host-uid2:enc-uid2,...]>\n\
```

Comment:
'>' is in wrong place

Reply:
Are you suggesting '>' should be before the closing square bracket? My reasoning about the current order had been that the text in the square brackets is optional. And the entire variable text sequence is closed by the '>'.